tests/unit/utils/parse-response-headers-test.js

```diff
@@ -0,0 +1,54 @@
+import parseResponseHeaders from 'ember-ajax/utils/parse-response-headers';
+import { module, test } from 'qunit';
+
+const CRLF = '\u000d\u000a';
+
+module('unit/adapters/parse-response-headers');
+
+test('returns {} when headersString is undefined', function(assert) {
+  let headers = parseResponseHeaders(undefined);
+
+  assert.deepEqual(headers, {}, '{} is returned');
+});
+
+test('header parsing', function(assert) {
+  let headersString = [
+    'Content-Encoding: gzip',
+    'content-type: application/json; charset=utf-8',
+    'date: Fri, 12 Feb 2016 19:21:00 GMT'
+  ].join(CRLF);
+
+  let headers = parseResponseHeaders(headersString);
+
+  assert.equal(headers['Content-Encoding'], 'gzip', 'parses basic header pair');
+  assert.equal(headers['content-type'], 'application/json; charset=utf-8', 'parses header with complex value');
+  assert.equal(headers.date, 'Fri, 12 Feb 2016 19:21:00 GMT', 'parses header with date value');
+});
+
+test('field-name parsing', function(assert) {
+  let headersString = [
+    'value-with-leading-space: value with leading whitespace',
+    'value-without-leading-space:value without leading whitespace',
+    'value-with-colon: value with: a colon',
+    'value-with-trailing-whitespace: banana'
```

Comment:
The key implied that there should be trailing whitespace on the value, but it doesn't look like there is here.  Am I missing something?